department-of-veterans-affairs/va.gov-cms
# VAGOV-2955: Change the view json tab to node view.

This mr replaces confusing View json tab data view with standard drupa node view implementation.
Task: https://va-gov.atlassian.net/browse/VAGOV-2850 / 2955

**QA:**

- Visit any node
- Visually verify that "View Json" tab now reads as "View"
- Verify that clicking view tab return node view
